README.md

```diff
@@ -12,22 +12,24 @@ Search provides a search module for CakePHP applications.
 The master branch has the following requirements:
 
 * CakePHP 3.0.0 or greater.
+* Patience
 
 ## Installation
 
 * Install the plugin with composer from your CakePHP Project's ROOT directory
 (where composer.json file is located)
-```sh
-php composer.phar require friendsofcake/search
+```shell
+php composer.phar require friendsofcake/search:~1.2
```

Comment:
There's no need to specify version. Composer will automatically use latest release and set version constraints in composer.json